R/Italy.R

```diff
@@ -0,0 +1,72 @@
+#' R6 Class containing country specific attributes and methods
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for Italy.
+#'
+#' @details Inherits from `dataClass`
+#'
```

Comment:
Usage example (helpful for testing works) wrap in \dontrun{} to avoid in running on CRAN etc.